src/MyEventEmitter.js

```diff
@@ -1,12 +1,69 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+
+    this.events[eventName].push(listener);
+  }
+
+  once(eventName, listener) {
+    const onceWrapper = () => {
+      listener();
+      this.off(eventName, onceWrapper);
+    };
+
+    this.on(eventName, onceWrapper);
+  }
+
+  off(eventName, listener) {
+    if (!this.events[eventName]) {
+      return;
+    }
+
+    this.events[eventName].filter(
+      eventListener => eventListener !== listener
+    );
```

Comment:
Filter doesn't change the initial array

Reply:
Yup, looks like you forgot to assign `this.events[eventName] = `